corehq/apps/locations/models.py

```diff
@@ -94,15 +94,21 @@ class LocationType(models.Model):
         null=True,
         related_name='+',
         db_column='expand_from',
-        on_delete=models.CASCADE,
+        on_delete=models.RESTRICT,
     )  # levels below this location type that we start expanding from
     _expand_from_root = models.BooleanField(default=False, db_column='expand_from_root')
     expand_to = models.ForeignKey(
         "self",
         null=True,
         related_name="+",
-        on_delete=models.CASCADE,
+        on_delete=models.RESTRICT,
     )  # levels above this type that are synced
+    restrict_cases_to = models.ForeignKey(
+        "self",
+        null=True,
+        related_name="+",
```

Comment:
https://docs.djangoproject.com/en/4.1/ref/models/fields/#django.db.models.ForeignKey.related_name
I hadn't seen this setting before.  Cool.